File: src/main/java/com/johnson/Hello.java
package com.johnson;

import java.sql.SQLOutput;

public class Hello {
    public static void main(String[] args) {
       /*System.out.println("Hello world");
       new Person().hello();*/
        Person p = new Person("Johnson" ,75.5,1.77);
        p.name="Johnson";
        p.hello();
        /*p.weight=75.5;
        p.height=1.77;*/
        System.out.println("Your BMI :" + p.bmi() );

    }
}

File: src/main/kotlin/com/johnson/Hello.kt
package com.johnson

fun main() {
//    println("Hello Kotlin")
//    Human().hello()
    val h = Human(name = "johnson" ,height = 1.77f, weight = 75.5f)
    h.hello()
    println(h.bmi())
}

class Human( var height : Float , var weight : Float, var name : String =""){
    init {
        print("Johnson $height $weight")
    }
//    constructor(height: Float,weight: Float,name:String):this(height, weight)
    fun bmi() : Float{
        val bmi = weight/(height*height)
        return bmi
    }

    fun hello(){
        println("Hello Kotlin")
    }
}